src/fides/api/ops/service/saas_request/override_implementations/firebase_auth_request_overrides.py

```diff
@@ -40,13 +42,23 @@ def firebase_auth_user_access(  # pylint: disable=R0914
     if identity == "email":
         emails = input_data.get("email", [])
         for email in emails:
-            user = auth.get_user_by_email(email, app=app)
-            processed_data.append(_user_record_to_row(user))
+            try:
+                user = auth.get_user_by_email(email, app=app)
+                processed_data.append(_user_record_to_row(user))
+            except UserNotFoundError:
+                logger.warning(
+                    f"Could not find user with email {Pii(email)} in firebase"
+                )
     elif identity == "phone_number":
         phone_numbers = input_data.get("phone_number", [])
         for phone_number in phone_numbers:
-            user = auth.get_user_by_phone_number(phone_number, app=app)
-            processed_data.append(_user_record_to_row(user))
+            try:
+                user = auth.get_user_by_phone_number(phone_number, app=app)
```

Comment:
hmmm. that may be a good edge case, good call out. i do think the phone number should be somewhat normalized by the time it gets here since it's presumably been entered through the privacy center UI and the whole DSR framework? not sure if that totally protects us here, i'd need to do some more testing.

that being said, would prefer to address in a follow up because this wouldn't be a regression with this change, and this change does give us important error handling (and it's a bit time sensitive because of that).